docker/venice-controller/single-dc-configs/controller.properties

```diff
@@ -0,0 +1,37 @@
+admin.port=5555
```

Comment:
a thought for future versions: 
In order to have a single point of change: we should try adding the repeated common configs like ports in a separate file/env (eg: ${ZOOKEEPER_ADDRESS}) and reuse them/populate them to these files on the fly. Will enable easy modification of them if/when needed.

Reply:
Won't do